app/src/components/TheDrawer.vue

```diff
@@ -1,149 +1,69 @@
 <template>
   <div
-    class="flex flex-col items-center menu pt-28 w-80 min-h-full bg-white text-base-content fixed"
+    class="flex flex-col items-center menu pt-28 w-80 min-h-full bg-white text-base-content fixed px-6 gap-3"
   >
-    <div class="flex flex-row items-center justify-center card w-64 h-24 bg-base-100">
+    <div class="w-full flex flex-row justify-start gap-4 card bg-base-100 px-5 py-4">
```

Comment:
Here the parent have a fixed width, so you don't need to fixe the width of the children. 
That why i use `w-full`

If you have a component like this it better 
- to have a padding for the card` px-5 py-4`
- then use gaps to seperate items and `gap-4`
- then justyfy it start. 
SO either you have the parent wide or small you will have the right height and the right width.
![image](https://github.com/globe-and-citizen/cnc-portal/assets/28951070/d02621a7-cd25-43a0-961f-4f3e4de9bf75)